widget/splitcontainer.go

```diff
@@ -33,46 +33,24 @@ func NewVSplitContainer(top, bottom fyne.CanvasObject) *SplitContainer {
 
 func newSplitContainer(horizontal bool, leading, trailing fyne.CanvasObject) *SplitContainer {
 	s := &SplitContainer{
+		Offset:     0.5, // Sensible default, can be overriden with SetRatio
```

Comment:
Oh sorry, close but this doc is out of date now.

Reply:
Fixed